tests/unit/linkry.test.ts

```diff
@@ -48,33 +49,74 @@ smMock.on(GetSecretValueCommand).resolves({
   SecretString: secretJson,
 });
 
-const testJwt = createJwt(undefined, "0", "test@gmail.com");
+const adminJwt = createJwt(undefined, ["LINKS_ADMIN"], "test@gmail.com");
 
-test("Happy path: Fetch all linkry redirects with proper roles", async () => {
+beforeEach(() => {
+  ddbMock.reset();
+});
+// Get Link
+
+test("Happy path: Fetch all linkry redirects with admin roles", async () => {
   ddbMock.on(QueryCommand).resolves({
     Items: [],
   });
 
   ddbMock
     .on(ScanCommand)
     .resolvesOnce({
-      Items: [],
+      Items: dynamoTableData,
     })
     .rejects();
 
   const response = await app.inject({
     method: "GET",
     url: "/api/v1/linkry/redir",
     headers: {
-      Authorization: `Bearer ${testJwt}`,
+      Authorization: `Bearer ${adminJwt}`,
     },
   });
 
   expect(response.statusCode).toBe(200);
+  let body = JSON.parse(response.body);
+  expect(body.ownedLinks).toEqual([]);
+});
+
+test("Happy path: Fetch all linkry redirects with admin roles owned", async () => {
+  const adminOwnedJwt = createJwt(
+    undefined,
+    ["LINKS_ADMIN"],
+    "bob@illinois.edu",
+  );
+
+  ddbMock.on(QueryCommand).resolves({
+    Items: dynamoTableData,
+  });
+
+  ddbMock
+    .on(ScanCommand)
```

Comment:
Minor nit: We should provide a second param to QueryCommand to make sure the right table is queried:

```
ddbMock.on(ScanCommand, {TableName: genericConfig.XYZ})
```